conda.recipe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set sdata = load_setup_py_data() %}
+
+package:
+  name: spatialpandas
+  version: {{ sdata['version'] }}
+
+source:
+  path: ..
+
+build:
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    {% for group,epoints in sdata.get("entry_points",{}).items() %}
+      {% for entry_point in epoints %}
+      - {{ entry_point }}
+      {% endfor %}
+      {% endfor %}
+
+requirements:
+  build:
```

Comment:
This should be `host` in modern-day conda.

Reply:
:+1: